master/internal/stream/system.go

```diff
@@ -94,7 +95,8 @@ func newDBListener(channel string) (*pq.Listener, error) {
 // NewPublisherSet constructor for PublisherSet.
 func NewPublisherSet() *PublisherSet {
 	return &PublisherSet{
-		Trials: stream.NewPublisher[*TrialMsg](),
+		PgAddress: "postgresql://postgres:postgres@localhost/determined?sslmode=disable",
```

Comment:
nit (non-blocking): is there a reason we aren't using `Address` to be consistent?

Reply:
honestly, don't remember why i started using `PgAddress`, but I think I'm more partial towards `DBAddress` now. I think `Address` is vague for the publisher